de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/AuthTestResourceTest.java

```diff
@@ -0,0 +1,176 @@
+		assertEquals("Admin can access admin only ressource", expectedResponse3, serverResponse3);
+		
+		String serverResponse4 = webTarget.path("/auth").path("/user_only")
+				.request()
+				.header(HttpHeaders.AUTHORIZATION, encodedAdmin)
+				.get()
+				.toString();
+		String expectedResponse4 = "InboundJaxrsResponse{context=ClientResponse{method=GET, uri=http://localhost:8000/rest/auth/user_only, status=403, reason=Forbidden}}";
+		assertEquals("Admin can't access user only ressource", expectedResponse4, serverResponse4);
+	}
+	
+	@Test
+	/**
+	 * Ignore repositories for now
+	 */
+	public void testRepositoryAuthorization() {
+		
+		String encodedUserNoRepo = DEFAULT_AUTHORIZATION_HEADER;
+		String serverResponse = webTarget.path("/auth").path("/repository").path("/testRepo")
+				.request()
+				.header(HttpHeaders.AUTHORIZATION, encodedUserNoRepo)
+				.get()
+				.toString();
+		String expectedResponse = ("InboundJaxrsResponse{context=ClientResponse{method=GET, uri=http://localhost:8000/rest/auth/repository/testRepo, status=403, reason=Forbidden}}");
+		assertEquals("This user can't access the repository", expectedResponse, serverResponse);
+		
+		String encodedUserWithRepo = BASIC_SCHEME + " " + Base64.getEncoder().encodeToString(USER_WITH_REPO.getBytes());
+		String serverResponse2 = webTarget.path("/auth").path("/repository").path("/testRepo")
+				.request()
+				.header(HttpHeaders.AUTHORIZATION, encodedUserWithRepo)
+				.get()
+				.toString();			
+		String expectedResponse2 = ("InboundJaxrsResponse{context=ClientResponse{method=GET, uri=http://localhost:8000/rest/auth/repository/testRepo, status=200, reason=OK}}");
+		assertEquals("This user can access the repository", expectedResponse2, serverResponse2);
+		
+		String encodedAdmin = BASIC_SCHEME + " " + Base64.getEncoder().encodeToString(ADMIN.getBytes());
+		String serverResponse3 = webTarget.path("/auth").path("/repository").path("/testRepo")
+				.request()
+				.header(HttpHeaders.AUTHORIZATION, encodedAdmin)
+				.get()
+				.toString();			
+		String expectedResponse3 = ("InboundJaxrsResponse{context=ClientResponse{method=GET, uri=http://localhost:8000/rest/auth/repository/testRepo, status=200, reason=OK}}");
+		assertEquals("Admins can access all repositories", expectedResponse3, serverResponse3);
+		
+		String serverResponse4 = webTarget.path("/auth").path("/repository").path("/testRepo").path("/property")
+				.request()
+				.header(HttpHeaders.AUTHORIZATION, encodedUserWithRepo)
+				.get()
+				.toString();			
+		String expectedResponse4 = ("InboundJaxrsResponse{context=ClientResponse{method=GET, uri=http://localhost:8000/rest/auth/repository/testRepo/property, status=200, reason=OK}}");
+		assertEquals("This user can access the repository property", expectedResponse4, serverResponse4);
+	}
+	
+	@Test(expected = Test.None.class /* No Exception expected */)

```

Comment:
Why specify this? Tests pass without exception by default

Reply:
Will be removed